app/scripts/directives/addMobileClient.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+(function() {
+  angular.module('openshiftConsole').component('addMobileClient', {
+    controller: [
+      '$scope',
+      'APIService',
+      'DataService',
+      'NotificationsService',
+      AddMobileClient
+    ],
+    controllerAs: 'ctrl',
+    bindings: {
+      onClose: '<',
+      project: '<',
+      serviceName: '<'
+    },
+    templateUrl: 'views/directives/add-mobile-client.html'
+  });
+
+  function AddMobileClient($scope,
+                       APIService,
+                       DataService,
+                       NotificationsService) {
+    var ctrl = this;
+    var mobileclientVersion = {
+      group: "mobile.k8s.io",
+      version: "v1alpha1",
+      resource: "mobileclients"
+    };
+    var context = {namespace: ctrl.project.metadata.name};
+
+    ctrl.$onInit = function() {
+      DataService.list(mobileclientVersion, context)
```

Comment:
@sedroche @maleck13 I think we should show all clients in the row and pop-ups. We could have a checkbox `apply overview filter` but it might be out of scope for these tickets.

Reply:
I think we should show all clients rather then filtering . For almost everyone they will likely have very few client definitions